src/main/java/fi/otavanopisto/kuntaapi/server/rest/OrganizationsApiImpl.java

```diff
@@ -1329,17 +1330,38 @@ public Response findOrganizationContact(String organizationIdParam, String conta
     
     return createNotFound(NOT_FOUND);
   }
-
+  
   @Override
-  public Response listOrganizationContacts(String organizationIdParam, Request request) {
+  public Response listOrganizationContacts(String organizationIdParam, String search, String sortByParam, String sortDirParam,
+      Long firstResult, Long maxResults, Request request) {    
+    Response validateResponse = validateListLimitParams(firstResult, maxResults);
+    if (validateResponse != null) {
+      return validateResponse;
+    }
+    
+    ContactSortBy sortBy = resolveContactSortBy(sortByParam);
+    if (sortBy == null) {
+      return createBadRequest(INVALID_VALUE_FOR_SORT_BY);
+    }
+    
+    System.out.println("Sort by:" + sortBy);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)